app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<% provide :title, "Resend confirmation" %>
+<h1 class="center">Resend confirmation instructions</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
```

Comment:
`resource` ở đây là gì anh 
tại sao phải khai báo resource

Reply:
1. resource : Đại diện cho model mà mình đã cấu hình để devise quản lý. Cụ thể ở đây là model User.
   +ex: 
       build_resource(sign_up_params) # will be the similar to user = User.new(sign_up_params)
       resource # will be an instance of User
       resource.valid? # will be the name as user.valid?
2. Tại sao phải khai báo resource : "Resource" is how developers want to refer to their "User" Model
  + Devise được tạo ra để làm việc với bất cứ model nào, không bắt buộc phải là model User
  + Nếu ta có thêm 1 model sử dụng devise ta có thể tái sử dụng lại: app/views/devise/confirmations/new.html.erb mà không phải tạo mới.
=> mục đích có thể áp dụng 1 devise view cho nhiều model và là chuẩn chung mà tác giả của gem này yêu cầu sử dụng để tham chiếu đến Model mà mình muốn sử dụng